### PR TITLE
Add file SignIn

### PR DESCRIPTION
**Pull request by ChmyrRoma**

### CODE REVIEW

File: src/components/Header/SignIn/SignIn.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Grid, Button, Input } from '@material-ui/core';
+import { Grid} from '@material-ui/core';
```
Comment (RomaGorshkov) [inline src/components/Header/SignIn/SignIn.jsx:2]: { Grid }

File: src/components/Header/SignIn/SignIn.jsx [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+import { Grid, Button, Input } from '@material-ui/core';
+import style from './SignIn.module.css';
+import doctor from '../../../assets/images/doctor.png'
 
 const SignIn = () => {
     return (
-        <div>SignIn</div>
+        <div>
+            <Grid className={style.headerSingIn}>
+                <h1>Империя здоровья</h1>
```
Comment (RomaGorshkov) [inline src/components/Header/SignIn/SignIn.jsx:10]: edit in english

File: src/components/Header/SignIn/SignIn.jsx [hunk 3]
```diff
@@ -1,8 +1,25 @@
...
+                <img src={doctor} />
+            </Grid>
+            <Grid container direction='column' alignItems='center' className={style.bodySignIn}>
+                <Grid container direction='column' alignItems='center' className={style.bodyInput}>
+                    <input placeholder='Логин' />
```
Comment (RomaGorshkov) [inline src/components/Header/SignIn/SignIn.jsx:15]: edit in english

File: src/components/Header/SignIn/SignIn.jsx [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
+                    <input placeholder='Пароль' />
```
Comment (RomaGorshkov) [inline src/components/Header/SignIn/SignIn.jsx:16]: edit in english

File: src/components/Header/SignIn/SignIn.jsx [hunk 5]
```diff
@@ -1,8 +1,25 @@
...
+                </Grid>
+                <Grid>
+                    <button className={style.bodyButton}><h3>Войти</h3></button>
```
Comment (RomaGorshkov) [inline src/components/Header/SignIn/SignIn.jsx:19]: edit in english

File: src/components/Header/SignIn/SignIn.jsx (hunk 3 above)
Comment (ChmyrRoma) [reply src/components/Header/SignIn/SignIn.jsx:15]: fixed

File: src/components/Header/SignIn/SignIn.jsx (hunk 5 above)
Comment (ChmyrRoma) [reply src/components/Header/SignIn/SignIn.jsx:19]: fixed

**Pull request merged**